### ISSUE TITLE
Class cast exception: java.lang.Double can not be cast to java.lang.Long (DevicePixelRatio)

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/ashotwrapper#8]
@huyen.le reported a defect of AShotWrapper v0.2.1

https://forum.katalon.com/t/webui-take-area-screenshot-as-checkpoint/47371/18

**Comment by kazurayam** [comment]
v0.2.1 AShotWrapper.java has this code:

```
    public static class DevicePixelRatioResolver {

        private DevicePixelRatioResolver() {}

        public static float resolveDPR(WebDriver driver) {
            JavascriptExecutor js = (JavascriptExecutor)driver;
            Long value = (Long)js.executeScript("return window.devicePixelRatio;");
            return (float)value;
        }
    }
```

This code is wrong.

**Comment by kazurayam** [comment]
A quote from 
https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio#value

>A double-precision floating-point value indicating the ratio of the display's resolution in physical pixels to the resolution in CSS pixels. 

The returned value could be a decimal floating point number, for example, 2.5.

The value depends on the machine on which you are working on. I am working on a Macbook Air 2019 which has DPR=2.0. Others may have newer machine. They might have DPR=2.5

As my DPR on my machine is 2.0, casting a Double to a Long would accidentally pass. However others may encounter an Exception saying "Class cast exception: java.lang.Double can not be cast to java.lang.Long"

**Comment by kazurayam** [comment]
fixed at v0.2.2
